dotcom-rendering/src/palette.ts

```diff
@@ -2438,6 +2477,36 @@ const cardBackgroundDark: PaletteFunction = ({ design, theme }) => {
 			return sourcePalette.neutral[0];
 	}
 };
+
+const onwardPlaceholderBackgroundLight: PaletteFunction = ({
```

Comment:
wondering if this should be used by Picture as well?

Reply:
I think we can take that as a follow-up, considering the size of this PR